cli/console.ts

```diff
@@ -45,6 +45,66 @@ export function question(questionText: string) {
   return prompt(questionText);
 }
 
+/**
+ * Test-friendly function that prompts the user for input. If running in a
+ * non-TTY (i.e. test) environment and DATAFORM_CLI_TEST_INPUTS is set,
+ * it consumes inputs from the environment variable.
+ * @param questionText The question text to display.
+ * @returns Users's input or test input.
+ */
+export function interactiveQuestion(questionText: string): string {
+  // If running in a non-TTY environment and test inputs are provided
+  if (!process.stdin.isTTY && process.env.DATAFORM_CLI_TEST_INPUTS !== undefined) {
+    const testInput = getTestInput(questionText);
+    if (testInput !== undefined) {
```

Comment:
I propose to simplify it, and throw from `getTestInput` if there is any error instead of returning undefined.